package.json

```diff
@@ -192,7 +192,7 @@
         "flow-copy-source": "^2.0.2",
         "form-serialize": "^0.7.2",
         "husky": "^1.1.2",
-        "immutable": "^3.7.4",
+        "immutable": "^3.8.1",
```

Comment:
Please update the corresponding entry in `peerDependencies` and add `BREAKING CHANGE` to the commit message, as well.